src/language-js/print/statement.js

```diff
@@ -177,7 +179,10 @@ function shouldPrintSemicolonAfterClassProperty(node, nextNode) {
   // this isn't actually possible yet with most parsers available today
   // so isn't properly tested yet.
   if (
-    (name === "static" || name === "get" || name === "set") &&
+    (name === "static" ||
```

Comment:
Can you add tests for all these names, and remove the comment above?

I think they can be tested except `static` fails on `flow` and `meriyah`.

Reply:
https://github.com/prettier/prettier/pull/13927